src/app/core-ui/main/status/status.component.spec.ts

```diff
@@ -1,23 +1,25 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
-import { StatusComponent } from './status.component';
+import { MdTooltipModule } from '@angular/material';
 
-import { ModalsModule } from '../../modals/modals.module';
-import { SharedModule } from '../../shared/shared.module';
 import { CoreModule } from '../../../core/core.module';
-import { MdTooltipModule } from '@angular/material';
+import { SharedModule } from '../../../wallet/shared/shared.module';
+import { ModalsModule } from '../../../modals/modals.module';
+import { CoreUiModule } from '../../core-ui.module';
+
+import { StatusComponent } from './status.component';
 
 describe('StatusComponent', () => {
   let component: StatusComponent;
   let fixture: ComponentFixture<StatusComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ StatusComponent ],
       imports: [
         SharedModule,
         CoreModule.forRoot(),
         ModalsModule,
+        CoreUiModule.forRoot(),
         MdTooltipModule
```

Comment:
tooltip already in CoreUi

Reply:
yes I need to remove all angular material stuffs from the specs as I already included CoreUiModule, the reason why I remain it due to other test cases which are reflected due to new imports.